### ISSUE TITLE
[Understanding/On Focus] Two wrong URLs

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#338]
_From @massic80 on June 11, 2018 15:55_

In https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html there are two wrong hashes:

- https://www.w3.org/TR/WCAG21/#dfn-user-interface-component should be https://www.w3.org/TR/WCAG21/#dfn-user-interface-components
- https://www.w3.org/TR/WCAG21/#dfn-changes-of-context should be https://www.w3.org/TR/WCAG21/#dfn-change-of-context

_Copied from original issue: w3c/wai-wcag-quickref#164_

**Comment by yatil** [comment]
Hi @massic80, I will move this issue to the repository (https://github.com/w3c/wcag) where it can be addressed :-)

**Comment by fstrr** [comment]
I'm not seeing this issue—I've clicked on the referenced links in Understanding document and they work, so I'm closing this. If this does need re-opening, please do so.

**Comment by massic80** [comment]
Hi @fstrr , the issue was on the URL fragments (it took several seconds to me to remember, after four years :) ): both the URLs reported here redirect to the same (correct) page, but if you click the second on any bullet point you are positioned to the correct chapter.
They must have been updated in the time, since they now redirect to another base URL, and work with the same fragments which were wrong before:
https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html#dfn-user-interface-component
https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html#dfn-changes-of-context

I don't know how the two pages
https://www.w3.org/WAI/WCAG21/Understanding/on-focus.htmland https://www.w3.org/TR/WCAG21/#dfn-user-interface-component tie together and what they have in common, but for the sake of consistency they should be aligned.

**Comment by fstrr** [comment]
Hey, @massic80. I'll take another look and see if I can work out what's going on and also see what other Understanding documents do in case there's some weird logic going on. Thanks for replying.

**Comment by fstrr** [comment]
I'm still not understanding this.

If I look at [Understanding On Focus](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html) and
1. click on the only User Interface Component link on the page, it links to a definition on the bottom of that page.
2. click on either of the two Change Of Context links on the page, they both take me to a definition on the bottom of that page.

For comparison, if I look at [Understanding Non-Text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html), the links to definitions do the same thing—link to definitions on the bottom of the page.

If there are incorrect links in the Understanding On Focus document, could you screenshot them for me—that'll probably be clearer. If I'm mis-understanding you and this has been fixed and can be re-closed, that would also be good to know.
